src/Icons/AdjustableBase.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+
+interface Props {
+  width?: string | number
+  height?: string | number
+  fillColor?: string
+  [x: string]: unknown // for the rest property
+}
+
+const AdjustableBase: React.FunctionComponent<Props> = ({
+  width = '24',
+  height = '15',
+  fillColor = '#2D2926',
+  ...rest
+}) => (
+  <svg xmlns="http://www.w3.org/2000/svg" width={width} height={height} {...rest} fill="none" viewBox="0 0 24 15">
```

Comment:
you're probably going to hate me....

I would put the `...rest` at the end (after the viewBox) so that we can override anything. Right now, we cannot override fill and viewBox. Also I would consider removing the `xmlns` on all the svgs

Reply:
_yaaaay_

I think I can have that done by next week 😆